beeline/src/java/org/apache/hive/beeline/logs/BeelineInPlaceUpdateStream.java

```diff
@@ -0,0 +1,65 @@
+package org.apache.hive.beeline.logs;
+
+import org.apache.hadoop.hive.common.log.ProgressMonitor;
+import org.apache.hive.jdbc.logs.InPlaceUpdateStream;
+import org.apache.hive.service.rpc.thrift.TJobExecutionStatus;
+import org.apache.hive.service.rpc.thrift.TProgressUpdateResp;
+
+import java.io.PrintStream;
+import java.util.List;
+
+public class BeelineInPlaceUpdateStream implements InPlaceUpdateStream {
+  private org.apache.hadoop.hive.common.log.InPlaceUpdate inPlaceUpdate;
```

Comment:
not required now, earlier had same class name for interface that this class extended